src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Figure;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    public static void main(String[] args) {
+
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
```

Comment:
Please use constant for magic value
```suggestion
        Figure[] figures = new Figure[FIGURES_AMOUNT];
```